File: CiscoPCCE.Toolkit/Bean/TtsServerXMLList.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("ttsServers")]
    public class TtsServerXMLList : BaseApiBean
    {
        [XmlElement("ttsServer")]
        public List<TtsServer>? Items { get; set; }
    }
}

File: JavaSrc/toolkit/bean/TtsServerXMLList.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="ttsServers")
public class TtsServerXMLList extends BaseApiBean {
  private List<TtsServer> items;

  @XmlElement(name="ttsServer")
  public List<TtsServer> getItems() {
     return this.items;
  }

  public void setItems(List<TtsServer> items) {
     this.items = items;
  }


}
